tools/auth.js

```diff
@@ -1120,7 +1120,23 @@ exports.loginWithTokenOrOAuth = function (conn, accountsConfiguration,
 
   // Either we didn't have an existing token, or it didn't work. Do an
   // OAuth flow to log in.
-  var redirectUri = url + '/_oauth/meteor-developer?close';
+  var redirectUri = url + '/_oauth/meteor-developer';
```

Comment:
we didn't, it was done in https://github.com/meteor/meteor/commit/95e25f895949aba29abfb0a3adf10e5f124597ac, see my comment above. We needed it to fix a communication failure to build farms